client/react/frontpage/components/StreamBar.js

```diff
@@ -1,39 +1,50 @@
 // StreamBar.js
 
 import React from 'react';
+
+// Bootstrap Elements
 import { Grid, Glyphicon, Collapse } from 'react-bootstrap';
+
+// Open-Source Components
 import Slider from 'react-slick';
+
+// In house components
 import ChatBox from './ChatBox';
-import isMobile from './misc/isMobile';
-import './StreamBar.scss';
+import StreamMarquee from './StreamMarquee';
+
+//mobile?
+import isMobile from './misc/isMobile.js';
 
-const trackURL =
+// styling
+require('./StreamBar.scss');
+
+var trackURL =
   'https://ws.audioscrobbler.com/2.0/?method=user.getRecentTracks&user=uclaradio&api_key=d3e63e89b35e60885c944fe9b7341b76&limit=10&format=json';
-const streamURL = 'http://uclaradio.com:8000/;';
-let stream;
+var streamURL = 'http://uclaradio.com:8000/;';
```

Comment:
Please use `const` and `let` over `var`. I think this happens in multiple files.